### DISCUSSION TITLE
<title>Problem with package r-base and libicu in termux

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40215] [Projects and Issues]
### Select Topic Area

Bug

### Body

CANNOT LINK EXECUTABLE "/data/data/com.termux/files/usr/lib/R/bin/exec/R": library "libicuuc.so.69" not found: needed by main executable

Is there a solution for that? Assuming I don't need to compile R with libicu 69. Thanks.

**Comment by ossanna16** [comment]
Hi there @rafaelfurt89 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
